client/src/components/itemCard.tsx

```diff
@@ -45,17 +47,27 @@ export const ItemCard = ({ item, onClick }: Props) => {
         })}
         onClick={handleClick}
       >
-        <Image
-          priority
-          src={item.image}
-          alt={`select ${
-            "first_name" in item
-              ? `${item.first_name} ${item.last_name}`
-              : item.name
-          }`}
-          width={200}
-          height={280}
-        />
+        {item.image ? (
+          <Image
+            priority
+            src={item.image}
+            alt={`select ${
+              "first_name" in item
+                ? `${item.first_name} ${item.last_name}`
+                : item.name
+            }`}
+            width={200}
+            height={280}
+          />
+        ) : (
+          <Image
+            priority
+            src="/assets/images/default-avatar-dark.svg"
```

Comment:
i think we are currently focusing on dark mode, but in the future, this could be dynamic, along with the `alt` tag, and could be filled in from whatever mode the user is using.